Add Hover Visual States to the Column Header. (#588)
telerik/maui-docs: controls/datagrid/columns/header.md

> ---

Reviewer: Please check the [first heading](https://github.com/telerik/maui-docs/blob/a24d52c04cadfe514bf36d5259864bd5ba3e5a80/controls/datagrid/columns/header.md#changing-the-text-in-the-header) in the article and edit them by implementing [title-case capitalization](https://docs.telerik.com/style-guide/titles-and-headings#capitalization).

title: Column Headers
page_title: .NET MAUI DataGrid Documentation - Columns Header

> description: Review the Column Header article to learn how to style and customize the text in the Telerik UI for .NET MAUI DataGrid Header.

Reviewer: ```suggestion
description: Learn how to style and customize the text in the Telerik UI for .NET MAUI DataGrid Header by using the properties of the component.
```

![.NET MAUI DataGrid Column Header Template](../images/header-content-template.png)

> ## Hover Visual State on Column Header

Reviewer: ```suggestion
## Color on Hover
```

> You can change the background color of the column header on hover visual state by setting the `BackgroundColor` property.

Reviewer: ```suggestion
You can change the hover state background color of the column header by setting the `BackgroundColor` property.
```

> This example demonstrates how to apply `BackGroundColor` to the DataGrid Header on Hover Visual State: 

Reviewer: ```suggestion
The following example demonstrates how to apply the `BackGroundColor` property to the DataGrid header for its hover visual state: 
```

```XAML

>  <VisualStateManager.VisualStateGroups>

Reviewer 2: Create sdk example for this case or include the approach in existing example inside the styling section. then add the code snippet to the documentation.
